src/DotNetProjectFile.Analyzers/DotNetProjectFile.Analyzers.csproj

```diff
@@ -74,7 +74,10 @@ v1.6.0
 - Proj0502: Fixed missing package name when reference was using Update instead of Include. (BUG)
 - Proj0503: Package license is unknown. (NEW RULE)
 - Proj0504: Package license has changed. (NEW RULE)
-      ]]>
+v1.5.13
+- Proj0040: Run NuGet security audits on all. (NEW RULE)
```

Comment:
```suggestion
- Proj0040: Run NuGet security audits on all packages. (NEW RULE)
```

Reply:
I'm not sure I think that is an improvement?! but someiting like all, including transient feels to long, what do you think?

Reply:
on all transative dependencies

Reply:
on all (transative) dependencies